isis/src/base/objs/RestfulSpice/RestfulSpice.cpp

```diff
@@ -120,7 +120,7 @@ namespace Isis::RestfulSpice{
           {"spkQuality", spkQuality}
           });
       // @TODO check that json exists / contains what we're looking for
-      json out = spiceAPIQuery("getTargetStates", args, "POST");
```

Comment:
Just as a heads up -- the "POST" here was necessary for images with a lot of states.  On small images, "GET" works fine, but many times the kernel set will exceed the maximum size of GET requests.

Reply:
Curious, were you able to test the POST call with a heavy payload? ISIS tests that incurred getTargetStates and getTargetOrientations calls seemed to fail until the call method was changed to GET.

Reply:
There's a [sibling PR](https://github.com/DOI-USGS/SpiceQL/pull/37) on SpiceQL that includes fixes necessary to get this working. I'm not sure how to easily tie those changes into CI, though.